### ISSUE TITLE
Loading... instead of edit

### ISSUE THREAD

**Issue by lauracrouse** [Azure/LogicAppsUX#4225]
### Describe the Bug with repro steps 

1. Open the Logic App 
2. Create a new app 
3. Add the "When an HTTP request is received." trigger 
4. Click on the trigger to modfiy. 

### What type of Logic App Is this happening in? 

Standard (Portal)

### Are you using new designer or old designer 

New Designer

### Did you refer to the TSG before filing this issue? [https://aka.ms/lauxtsg](<https://aka.ms/lauxtsg>) 

Yes

### Workflow JSON 

```json
{
    "definition": {
        "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
        "actions": {},
        "contentVersion": "1.0.0.0",
        "outputs": {},
        "triggers": {
            "When_a_HTTP_request_is_received": {
                "inputs": {
                    "method": "POST"
                },
                "kind": "Http",
                "type": "Request"
            }
        }
    },
    "kind": "Stateless"
}
```

### Screenshots or Videos

![image](<https://github.com/Azure/LogicAppsUX/assets/159064616/da4b4602-522a-4e60-a993-c0ddca027e02>)

![image](<https://github.com/Azure/LogicAppsUX/assets/159064616/8ec54ec3-3861-4127-8a3f-baf822d3e91e>)

### Browser

Edge

### Additional context

I found that I CAN edit it in the older version. This is definitely a bug!

**Comment by nidubey1** [comment]
Hi @takyyon , the Microsoft Support had been reached out with another customer issue facing similar error. In this scenario the issue occurs for any edit in any workflow where schema need to be provided such as parse JSON/Request trigger. Attaching the impacted workflow.json, and it is only occurring in the customer Logic Apps and not reproducible in our internal application. Session ID for the customer issue scenario: 29eab2613da540f384e3db4039e35f10
[workflow 2.json](https://github.com/Azure/LogicAppsUX/files/14404151/workflow.2.json)

**Comment by hartra344** [comment]
This was caused by us loading the Monaco editor(used for editing JSON and javascript actions like this) from the CDN rather than bundling it directly with the application. This works in most scenarios but does break if you're no a network that doesn't allow access to the CDN. 
We have fixed this in #4401 , but it's took invasive of a change to hotfix. I'd advise using the old/fallback designer in the meantime if you can't access a more open network but it should be fixed in production in about a week or 2.
